### DISCUSSION TITLE
能设置目标的启动参数吗？

### DISCUSSION THREAD

**Discussion by Domain** [xmake-io/xmake#2518] [Q&A]
没找到相关文档，有这个功能吗？现在只能run后面自己加参数？

**Comment by waruqi** [comment]
目前只能自己 xmake run --xxx 加，或者自己配置 on_run 重写内部规则，自己调用 os.execv(target:targetfile(), args) 自己执行追加

**Reply by Domain** [reply]
希望能够加一个，也方便生成vs工程调试使用

**Reply by waruqi** [reply]
你可以提 feature requrest 过来。
